_includes/head.html

```diff
@@ -65,4 +65,5 @@
 
 <!-- Scripts
 ================================================== -->
-  <script src="{{ site.baseurl }}/assets/js/vendor/jquery-1.11.3.min.js"></script>
\ No newline at end of file
+  <script src="{{ site.baseurl }}/assets/js/vendor/jquery-1.11.3.min.js"></script>
```

Comment:
@msecret added to this PR: https://github.com/18F/usfwds/commit/849a3dde9cf1e66bb8b34613bfb5bc1b57be087a